protractor.conf.js

```diff
@@ -26,10 +26,13 @@ exports.config = {
     'chromeOptions': {
       'args': [
         '--headless',
-        '--disk-cache-dir=/dev/null',
+        '--no-sandbox',
+        '--disable-gpu',
+        '--disable-extension',
+        '--incognito',
         '--disable-web-security',
         '--window-size=1920,1080',
-        'lang=es-AR'
+        '--lang=es-AR'
```

Comment:
Is that meant to be an illegal language code? If so, I suggest that we use a language code that is not assigned to any language to avoid confusion.